test_pull_requests

```diff
@@ -1650,7 +1805,12 @@ elsif mark_test_success_pull_id
 elsif mark_testonlyextended_success_pull_id
   @api_client.mark_test_success(mark_testonlyextended_success_pull_id, pull_id_repo, Properties['settings']['testonlyextended_settings'])
 elsif local_merge_pull_id
-  @api_client.local_merge_pull_request(local_merge_pull_id, pull_id_repo)
+  if ENV['MERGE'].to_s.downcase == 'true'
```

Comment:
`.to_s` here will turn `nil` into the empty string. This test needs to happen for the local merge to succeed when a `test_pull_requests` job is called with `MERGE=true`, e.g. if the base commit has changed and the PR needs to be tested again.